src/Analyzers/CSharp/Tests/UseParameterNullChecking/UseParameterNullCheckingTests.cs

```diff
@@ -1394,21 +1394,41 @@ void local()
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseIsNullCheck)]
-        public async Task TestNotWithUserDefinedOperator()
```

Comment:
please don't remove existing tests.  if the behavior change, just update the expected results for it.

Reply:
oh.  you renamed it and git got confused n/m.

Reply:
Diff is misleading here. The test is still present below